low-level/files/opt/seagate/sspl/setup/sspl_prepare.py

```diff
@@ -27,6 +27,7 @@
 from cortx.utils.validator.v_network import NetworkV
 from files.opt.seagate.sspl.setup.setup_error import SetupError
 from framework.utils.utility import Utility
+from framework.utils.conf_utils import fetch_config
```

Comment:
This became unused as you removed its reference call.
Please check and remove it if nowhere used inside this module.